content/english/anti-vegan/humans-are-superior.md

```diff
@@ -0,0 +1,107 @@
+
+What most people forget (*or choose to overlook*) in using the term superior is that; *We alone* determine the ranking systems. *We alone* determine what denotes better quality. *We alone* determine what makes one thing more important than another. To presuppose that 'we' are superior to 'them' and to then *choose* an attribute that separates 'us' from 'them' as (somehow) demonstrating superiority is a fallacy. Identifying difference never identifies superiority.  
+
+> "For instance, on the planet Earth, man had always assumed that he was more intelligent than dolphins because he had achieved so much—the wheel, New York, wars and so on—whilst all the dolphins had ever done was muck about in the water having a good time. But conversely, the dolphins had always believed that they were far more intelligent than man—for precisely the same reasons."  
+> - Douglas Adams *from* Hitchhiker's Guide to the Galaxy, 1979
+
+
+
+### The flawed reasoning of Superiority and Domination
+
+> For any X and Y, if X is superior (or morally superior) to Y, then X is morally justified in subordinating Y.
+ 
+ or in plain English (and, as often used in anti-vegan rhetoric) 
+ 
+ > if humans are superior to nonhuman animals, then humans are morally justified in treating nonhuman animals as a resource. 
+ 
+
+This logic is used to claim that humans are superior to, and may dominate, nonhumans *due to some difference* that is elevated (by humans) in importance. However, without the "assumptions  that *humans are superior* to (at least some) nonhumans, and that *superiority justifies subordination*, all one has is some difference" - Karen J. Warren *from* The Power and the Promise of Ecological Feminism *in* Environmental Ethics.
+
+This creates what Warren refers to as an *oppressive conceptual framework*. An example is patriarchal thinking which falsely justifies the subordination of women by men based on nothing more than difference. Warren further claims that this logic of domination, and any conceptual framework which gives rise to it, "must be abolished  in order to make possible a meaningful notion of difference..."
+
+We need to move away from the oppressive conceptual framework that maintains *sameness* and only extends the moral community to those who are similar to us by application of some self-determined criteria. Warren refers to this as *arrogant perception*, resulting in a false notion of superiority which gives rise to the claim that those like us deserve moral consideration and others don't.
+
+An alternative *loving perception* "presupposes and maintains difference - a distinction between the self and other, between human and at least some nonhumans - in such a way that perception of the other as other *is* an expression of love for one who/which is recognised at the outset as independent, dissimilar, different."
```

Comment:
Maybe consider adding at this point a short plain English explanation of what a 'loving perception' looks like as an important reminder (aside from the quick response explanation) of what kindness and compassion for those who are different looks like. I appreciate these paragraphs are analytical but they are difficult to understand (to me with the average brain) and further explanation could be useful in cementing the ideas.

Reply:
Trying.   Ecofeminism is bleeding' difficult and I don't want to change the meaning by simplifying. ...but trying...

Reply:
Excellent! I really like your explanation and much better that anything I could have come up with. It's clear and simple enough for anyone to understand.